aspnetcore/fundamentals/websockets/samples/7.x/WebSocketsSample/Controllers/WebSocketController.cs

```diff
@@ -0,0 +1,47 @@
+using System.Net.WebSockets;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebSocketsSample.Controllers;
+
+// <snippet>
+public class WebSocketController : ControllerBase
+{
+    [HttpConnect("/ws")]
```

Comment:
Have both `[HttpConnect("/ws")]` and `[HttpGet("/ws")]`

Reply:
Why separate methods? The same code will go into both methods

Reply:
That's not my concern. What I'm saying is you need both `[HttpGet]` and `[HttpConnect]` on the same method otherwise you won't support both HTTP/1 and HTTP/2 websocket requests.

Reply:
I get it.  Sorry, I misunderstood.  Thanks Brennan!